dns/zones/workspace-variables/apply-zone.tfvars.json

```diff
@@ -0,0 +1,17 @@
+{
+  "hosted_zone": {
+    "apply-for-teacher-training.education.gov.uk" : {
+      "caa_records": {
+      },
+      "txt_records": {
+        "@": {
+          "value": "v=spf1 -all"
+        },
+        "_dmarc": {
+          "value": "v=DMARC1; p=reject; sp=reject; rua=mailto:dmarc-rua@dmarc.service.gov.uk; ruf=mailto:dmarc-ruf@dmarc.service.gov.uk"
+        }
+      },
+      "resource_group_name": "s189p01-applydomains-rg"
+    }
+  }
+}
```

Comment:
Missing CAA record?

Reply:
Yes - we decided we couldn't add one of these in just yet.